model/simulationtests/doas.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'openstudio'
+require_relative 'lib/baseline_model'
+
+model = BaselineModel.new
+
+# make a 1 story, 100m X 50m, 1 zone building
+model.add_geometry({ 'length' => 100,
+                     'width' => 50,
+                     'num_floors' => 1,
+                     'floor_to_floor_height' => 4,
+                     'plenum_height' => 0,
+                     'perimeter_zone_depth' => 0 })
+
+# add windows at a 40% window-to-wall ratio
+model.add_windows({ 'wwr' => 0.4,
+                    'offset' => 1,
+                    'application_type' => 'Above Floor' })
+# add thermostats
+model.add_thermostats({ 'heating_setpoint' => 24,
+                        'cooling_setpoint' => 28 })
+
+# assign constructions from a local library to the walls/windows/etc. in the model
+model.set_constructions
+
+# set whole building space type; simplified 90.1-2004 Large Office Whole Building
+model.set_space_type
+
+# add design days to the model (Chicago)
+model.add_design_days
+
+
+# In order to produce more consistent results between different runs,
+# we sort the zones by names
+zones = model.getThermalZones.sort_by { |z| z.name.to_s }
+
+controller = OpenStudio::Model::ControllerOutdoorAir.new(model)
+oas = OpenStudio::Model::AirLoopHVACOutdoorAirSystem.new(model, controller)
+doas = OpenStudio::Model::AirLoopHVACDedicatedOutdoorAirSystem.new(model, oas)
+airloop = OpenStudio::Model::AirLoopHVAC.new(model)
+
+oas.addToNode(airloop.supplyOutletNode)
+doas.addAirLoop(airloop)
+airloop.addBranchForZone(zone)
```

Comment:
The AirLoop has no fan, SPM, and the Zone is added but without an ATU. Is that normal?